guides/common/modules/proc_chainbooting-ipxe-from-pxelinux.adoc

```diff
@@ -44,13 +44,11 @@ If you want to change the default values in the template, clone the template and
 . From the *PXELinux template* list, select the template you want to use.
 . From the *iPXE template* list, select the template you want to use.
 . Click *Submit* to save the changes.
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*, and select the host you want to use.
+. In the {ProjectWebUI}, navigate to *Configure* > *Host Groups*, and select the host group you want to configure.
 . Select the *Operating System* tab.
+. Select the *Architecture* and *Operating system*.
 . Set *PXE Loader* to *PXELinux BIOS* to chainboot iPXE via PXELinux, or to *iPXE Chain BIOS* to load `undionly-ipxe.0` directly.
-. Select the *Templates* tab, and from the *PXELinux template* list, select *Review* to verify the template is the correct template.
-. From the *iPXE template* list, select *Review* to verify the template is the correct template.
-If there is no PXELinux entry, or you cannot find the new template, navigate to *Hosts* > *All Hosts*, and on your host, click *Edit*.
-Click the *Operating system* tab and click the Provisioning Template *Resolve* button to refresh the list of templates.
```

Comment:
If we change the steps above to the host group workflow, then these steps are redundant, so I removed them. Can we agree on this?

Reply:
Not only are they redundant, they aren't set up correctly. This templates tab is in host->Operating system, but there's no "review" option there AFAIK. Removing those steps are fine.